cmd/influx/main.go

```diff
@@ -249,6 +258,22 @@ func (c *CommandLine) SetFormat(cmd string) {
 	}
 }
 
+func (c *CommandLine) dump() {
```

Comment:
Would it make sense to change this to `executeDump` so we don't have `c.Dump` and `c.dump`?